### DISCUSSION TITLE
Compare line by line(from newbie)

### DISCUSSION THREAD

**Discussion by vuslatx** [WinMerge/winmerge#1244] [Q&A]
Hi.

I have two text files. They include "windows dir names".
For example:

**Text1:**
C:\Windows\Microsoft
C:\Windows\Google \\Common line
D:\apps\Chrome

**Text2:**
C:\Windows\Google \\Common line
D:\Desktop\Github

I want to export different lines. I don't want to see common lines.

**Exported File:**
C:\Windows\Microsoft
D:\apps\Chrome
D:\Desktop\Github
(Sorting is not important)

Note: Please explain like I'm five. How to configure WinMerge for to achieve this?

**Comment by sdottaka** [comment]
If you set the Diff Context to 0 Lines as shown below, only the differences will be displayed.
For both panes, select the text with Ctrl + A, copy it to the clipboard with Ctrl + C, and paste the text into another text editor to get results close to your expectations. 

![image](https://user-images.githubusercontent.com/98126/157772093-5bf4d857-97a6-403d-b5a3-bca7a1c26173.png)
